src/core/include/iDynTree/Core/Position.h

```diff
@@ -67,6 +67,11 @@ namespace iDynTree
          */
         Position(iDynTree::Span<const double> other);
 
+        /**
+         * Constructor from a raw buffer of 3 doubles.
+         */
+        Position(const double* in_data, const unsigned int in_size);
+
```

Comment:
If you need the function in the bindings I strongly suggest to not modify the C++ part but only the bindings.

Reply:
You are right reverted in 4a5509a1c946ac776242220e709b9e81b42f4a01

Reply:
Reverted in 4a5509a1c946ac776242220e709b9e81b42f4a01